src/envoy/mixer/integration_test/failed_request_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	rpc "github.com/googleapis/googleapis/google/rpc"
+)
+
+const (
+	mixerAuthFailMessage = "Unauthenticated by mixer."
+)
+
+// Check attributes from a fail GET request from mixer
+const checkAttributesMixerFail = `
```

Comment:
It seems that this is identical to `checkAttributesOkGet`. Should these two have some differences?